config/jobs/cert-manager/website/cert-manager-website-periodics.yaml

```diff
@@ -14,6 +14,7 @@ periodics:
   annotations:
     testgrid-dashboards: jetstack-cert-manager-website
     testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
+    testgrid-disable-prowjob-analysis: "true"
```

Comment:
This annotation's functionality exists after this PR: https://github.com/kubernetes/test-infra/pull/21842

Reply:
Does that mean we need to upgrade our Prow installation?

Reply:
No, I think we generate TestGrid config [here](https://github.com/jetstack/testing/blob/master/config/jobs/testing/testing-trusted.yaml#L77) and that was already pulling the latest version of the `transfigure` container. As far as I understand nothing else should be looking at that annotation.
(I did look at Prow upgrade this am, but looks like there might be some larger changes (Plank being removed? https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md#breaking-changes) so maybe that's a separate PR)